app/models/course.rb

```diff
@@ -56,6 +56,8 @@
 
 #= Course model
 class Course < ApplicationRecord
+  before_save { ensure_home_wiki_in_courses_wikis }
```

Comment:
This sould be in the `callbacks` section, and should be done with a symbol instead of a block to match the other callbacks.